docs/notebooks/image_to_image_arcface.ipynb

```diff
@@ -0,0 +1,362 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p8jc8EyfruKw"
+      },
+      "source": [
+        "# Image-to-Image Search with ArcFaceLoss\n",
+        "\n",
+        "<a href=\"https://colab.research.google.com/drive/1ZS9FmnR9FzO_JYGCPazFM7TcMNQl51xM?usp=sharing\"><img alt=\"Open In Colab\" src=\"https://colab.research.google.com/assets/colab-badge.svg\"></a>\n",
```

Comment:
The only notebook seems not to be up-to-date with this one. If I click on this link I get to a notebook with the title: "Image-to-Image Search via ResNet50".